### PR TITLE
docs(ui5-dialog): document usage of ui5-bar better

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
PartOf: #7354

### CODE REVIEW

File: packages/main/src/Dialog.ts [hunk 1]
```diff
@@ -85,6 +85,8 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <li>content - Used to style the content of the component</li>
  * <li>footer - Used to style the footer of the component</li>
  * </ul>
+ * <b>Note:</b> When ui5-bar is used in the Header or in the Footer additional styling is needed. The default Dialog's paddings should be removed.
```
Comment (BorisDafov) [inline packages/main/src/Dialog.ts:88]: I suggest to simplify:
<b>Note:</b> When a <code>ui5-bar</code> is used in the header or in the footer, you should remove the default dialog's paddings.
 * For more information see the sample "Bar in Header/Footer".

File: packages/main/src/Dialog.ts [hunk 2]
```diff
@@ -229,6 +231,8 @@ class Dialog extends Popup {
 	/**
 	 * Defines the header HTML Element.
 	 * <br><br>
+	 * When ui5-bar is used in the Header additional styling is needed. The default Dialog's paddings should be removed.
```
Comment (BorisDafov) [inline packages/main/src/Dialog.ts:234]: Respectively here:
* When a <code>ui5-bar</code> is used in the header, you should remove the default dialog's paddings.

File: packages/main/src/Dialog.ts [hunk 3]
```diff
@@ -242,6 +246,8 @@ class Dialog extends Popup {
 
 	/**
 	 * Defines the footer HTML Element.
+	 * <br><br>
+	 * When ui5-bar is used in the Footer additional styling is needed. The default Dialog's paddings should be removed.
```
Comment (BorisDafov) [inline packages/main/src/Dialog.ts:250]: same:
* When a <code>ui5-bar</code> is used in the footer, you should remove the default dialog's paddings.

**Review by BorisDafov — CHANGES_REQUESTED** [review]

**Review by BorisDafov — APPROVED** [review]

**Review by BorisDafov — APPROVED** [review]
I approve these changes

File: packages/main/src/Dialog.ts [hunk 4]
```diff
@@ -85,6 +85,10 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <li>content - Used to style the content of the component</li>
  * <li>footer - Used to style the footer of the component</li>
  * </ul>
+ * <b>Note:</b> When a <code>ui5-bar</code> is used in the header or in the footer, you should remove the default dialog's paddings.
+ * <br>
+ * For more information see the sample "Bar in Header/Footer".
```
Comment (dimovpetar) [inline packages/main/src/Dialog.ts:90]: Ask Ilhan if it's fine to refer a concrete sample from the JSDoc

**Review by dimovpetar — CHANGES_REQUESTED** [review]
In the sample https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-dialog--bar-in-dialog the styles to remove the paddings cannot be seen. Please check how the sample could show them

File: packages/main/src/Dialog.ts (hunk 4 above)
Comment (LidiyaGeorgieva) [reply packages/main/src/Dialog.ts:90]: He is OK with this :)

**Review by ilhan007 — APPROVED** [review]

**Comment by LidiyaGeorgieva** [comment]
> In the sample https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-dialog--bar-in-dialog the styles to remove the paddings cannot be seen. Please check how the sample could show them

The issue is in the HTML add-on. It is reported as comment here: https://github.com/SAP/ui5-webcomponents/issues/7284

**Review by dimovpetar — APPROVED** [review]

**Pull request merged**